build/checkOrphanedImages.js

```diff
@@ -0,0 +1,91 @@
+const path     = require('path');
+const fs       = require('fs');
+const common   = require('../scripts/common');
+const {
+  prompt,
+  flatten,
+  fetchFiles } = require('./utils/cli');
+
+const ignored_files            = [];
```

Comment:
@easteregg Also here it should be better to use `.gitignore` file as default value.